worker/docker_client.py

```diff
@@ -229,6 +229,35 @@ def download_image(self, docker_image, loop_callback):
                 loop_callback(status)
 
     @wrap_exception('Unable to start Docker container')
+    def run_nvidia_smi(self, args, docker_image):
```

Comment:
I think I can see why but wondering what your reasoning was for running 'nvidia-smi' inside Docker rather than directly?